packages/dev/inspector-v2/src/components/properties/nodes/abstractMeshProperties.tsx

```diff
@@ -1,4 +1,4 @@
-import type { FunctionComponent } from "react";
+import { useState, type FunctionComponent } from "react";

```

Comment:
Split this into a type import separate from the useState import.

Reply:
Fixed